src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -1262,15 +1264,19 @@ class DevTask extends AbstractFeatureTask {
 
         File buildFile = project.getBuildFile();
 
+        List<Path> webResourceDirs = LooseWarApplication.getWebSourceDirectoriesToMonitor(project);
         // Instantiate util before any child gradle tasks launched so it can help find available port if needed
+        List<ProjectModule> projectModules = getProjectModules()
```

Comment:
Can you add some comments here to explain the difference between projectModules and allProjects? And what they are going to be used for?

Reply:
ProjectModules are constructed passed to DevUtils for multi module inside DevUtils. its used in multiple areas for watching sub project files

Reply:
Maybe I can remove allProjects now, and add it later to avoid confusion on where its used